examples/transformers_text.py

```diff
@@ -243,6 +237,17 @@ def mean_pooling(last_hidden_state: Tensor, attention_mask) -> Tensor:
 val_loader = DataLoader(val_tensor_frame, batch_size=args.batch_size)
 test_loader = DataLoader(test_tensor_frame, batch_size=args.batch_size)
 
+if not args.finetune:
+    text_stype_encoder = LinearEmbeddingEncoder()
+else:
+    model_cfg = TextModelConfig(model=text_encoder, out_channels=768)
+    col_to_model_cfg = {
+        col_name: model_cfg
+        for col_name in train_tensor_frame.col_names_dict[
+            torch_frame.text_tokenized]
+    }
+    text_stype_encoder = LinearModelEncoder(col_to_model_cfg=col_to_model_cfg)
```

Comment:
Here we are using the same text_encoder instance for all columns. Just want to confirm if this is intended. Can this be a source of performance regression?

Reply:
I don't think so. I am benchmarking on a dataset with just one text column.

Reply:
Sounds good.